variables.tf

```diff
@@ -1,5 +1,103 @@
-#variable "example" {
-#  type        = string
-#  default     = ""
-#  description = "This is just an example of a variable"
-#}
+variable "org_account_mappings" {
+  type = list(object({
+    default_lacework_account = string
+    mapping = list(object({
+      lacework_account = string
+      aws_accounts     = list(string)
+    }))
+  }))
+  default     = []
+  description = "Mapping of AWS accounts to Lacework accounts within a Lacework organization"
+}
+
+variable "use_existing_iam_role" {
+  type        = bool
+  default     = false
+  description = "Set this to true to use an existing IAM role"
+}
```

Comment:
I am a little bit hesitant to allow users to do this because what happens if they create the
IAM role in the AWS Audit account, instead of the AWS Log Archive?

If we want to allow this, then we need to at least document that the IAM role has to exist
in that AWS account/provider.

Reply:
I see what you are saying.  Where could we document this appropriately?  In the README.md or are you talking about on the Lacework docs website?  I figure I will need to do a PR to the docs website to explain this module a bit anyway, so maybe that would be a good place to put this information.

Reply:
lets do it in the description for now, but in support.lacework.com we must document it clearly